### ISSUE TITLE
Parent selector and interpolation

### ISSUE THREAD

**Issue by DanPurdy** [tonyganch/gonzales-pe#116]
This seemed a separate issue to the interpolation issues I'd opened up. Hope that's ok.

This block is causing a parse error in 3.2.1

``` scss
/// Input placeholder styling
/// @param {List} $styles - Styles
/// @requires $kp-ph--prefixes
/// @group forms
@mixin kp-placeholder-styles($styles) {
  @each $pseudo in $kp-ph--prefixes {
    @at-root &#{$pseudo} {

      @each $key, $value in $styles {
        #{$key}: #{$value};
      }
    }

    @at-root &:focus#{$pseudo} {
      color: transparent;
    }
  }
}
```

the line in question is `@at-root &#{$pseudo} {` If i remove the ampersand it parses fine.

First reported [here](https://github.com/sasstools/sass-lint/issues/184)

**Comment by tonyganch** [comment]
Related  to #114.
Thanks!

**Comment by tonyganch** [comment]
Should be fixed with #137, will publish a release in a few hours.

**Comment by tonyganch** [comment]
Please check in `@3.2.3`.
